core/neat/functions/_grid-calc.scss

```diff
@@ -0,0 +1,21 @@
+@charset "UTF-8";
+/// Return a calulated column width that can then be passed in to a `calc()`.
+///
+/// @argument {map} $grid
+///
+/// @argument {number} $columns
+///
+/// @return {string}
+///
+/// @example scss
+///   _grid-calc($neat-grid, 4)
+///
+/// @access private
+
+@function grid-calc($grid, $columns) {
```

Comment:
If this function is private, as indicated on Line 13, it should probably be named `_grid-calc` (with an underscore).